File: org.summer.view.widget/src/org/summer/view/widget/core/Visibility.java
package org.summer.view.widget.core;
public enum Visibility //: byte
{ 
    /// <summary> 
    /// Normally visible.
    /// </summary> 
    Visible = 0,

    /// <summary>
    /// Occupies space in the layout, but is not visible (completely transparent). 
    /// </summary>
    Hidden, 

    /// <summary>
    /// Not visible and does not occupy any space in layout, as if it doesn't exist. 
    /// </summary>
    Collapsed
}

File: summer/WebContent/js/system/windows/Visibility.js
/**
 * Visibility
 */
define(["dojo/_base/declare", "system/Type"], function(declare, Type){
	
    var Visibility = declare("Visibility", Object, { 
    });	
    
    /// Normally visible.
    Visibility.Visible = 0;

    /// Occupies space in the layout, but is not visible (completely transparent). 
    Visibility.Hidden=1; 

    /// Not visible and does not occupy any space in layout, as if it doesn't exist. 
    Visibility.Collapsed=2;

//    Visibility.Type = new Type("Visibility", Visibility, [Object.Type]);
    return Visibility;
});
